wasm/wasm_source/src/vp_implicit.rs

```diff
@@ -0,0 +1,601 @@
+//! Default VP for implicit transparent addresses.
+//!
+//! This VP currently provides a signature verification against a public key for
+//! sending tokens (receiving tokens is permissive).
+//!
+//! It allows to bond, unbond and withdraw tokens to and from PoS system with a
+//! valid signature.
+//!
+//! It allows to fulfil intents that were signed by this account's key if they
+//! haven't already been fulfilled (fulfilled intents are added to the owner's
+//! invalid intent set).
+//!
+//! Any other storage key changes are allowed only with a valid signature.
```

Comment:
I think we don't want to give implicit accounts arbitrary storage as of my last recollection, but this is a design detail